modules/internal/ChapelTuple.chpl

```diff
@@ -434,7 +434,7 @@ module ChapelTuple {
   // ...followed by a more complicated/general case for heterogeneous
   // tuples because the result type is more complex to describe.
   //
-  inline proc +(a: _tuple, b: _tuple) {
+  inline operator +(a: _tuple, b: _tuple) {
```

Comment:
Tuples are weird in a lot of ways but I'd think most of these can/should be operator methods as well but that's probably best viewed as future work.